### PR TITLE
Switch Source Serif Pro to import from Google Fonts

### PR DESCRIPTION
**Pull request by iamrentman**
Switch Source Serif Pro to import from Google Fonts instead of hosted files.
- Update `_fonts.scss` with import copy/pasted from Google Fonts, remove hosted file imports.
- Run `publish` to generate compiled decanter.css file.

# READY FOR REVIEW

# Summary
Import all versions of Source Serif Pro from Google Fonts (instead of hosted AWS files) now that they are available from Google. This maintains consistency with how the other fonts are imported to the project.

# Needed By (Date)
No specific deadline.

# Urgency
Not critical

# Steps to Test
1. Create tests file in the `developer` directory – you'll need `test.twig` and `test.scss` for this
2. In `test.twig` add markup for at least one heading and some body text. 
3. Make a portion of each the heading and the body italic by adding an `<em>` or `<i>` to a few words.
4. In `test.scss` add a rule so your content displays in the Source Serif font – `font-family: $su-font-serif;`
5. `npm run dev` to compile the assets.
6. `npm run start` to start the development server and open your test page.
7. Check your test page in the browser to ensure that it displays the proper serif font – Source Serif Pro – and ensure that the italics display appropriately.
8. Using dev tools, view the `decanter.css` file in the Sources tab. Line 2 should now be a longer string that includes all variations of Source Serif Pro.

# Affected Projects or Products
Just a helpful update, that's all.

# Associated Issues and/or People
I've discussed this change in the #decanter Slack channel with @yvonnetangsu and @sherakama. We agreed that it'd be worthwhile. This is my first PR to Decanter, so lemme know if I missed something. :)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@iamrentman Yay 🎉  Thank you so much! I'll test this out later today 😄

File: core/src/scss/core/fonts/_fonts.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
 @import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,200i,300,300i,400,400i,600,600i,700,700i,900,900i');
```
[the remark this answers is not in the document]
Comment (sherakama) [reply core/src/scss/core/fonts/_fonts.scss:12]: Or does this need to go now that we have the one on line 15?

[the remark this answers is not in the document]
Comment (yvonnetangsu) [reply core/src/scss/core/fonts/_fonts.scss:12]: Ohh looks like this import needs to be changed from "Source Sans Pro" to "Source Serif Pro" @iamrentman 😃 And if you could please help us update all the import lines using Google's latest format as @sherakama suggested, that'd be awesome, e.g.
Screen Shot 2022-09-12 at 10 31 29 AM

[the remark this answers is not in the document]
Comment (yvonnetangsu) [reply core/src/scss/core/fonts/_fonts.scss:12]: I'm on the fence about the `display=swap` part - Google lighthouse favors it as it displays the text before the Google font gets load, but you do get a flash of ugly fonts which sometimes isn't that great either 😬

[the remark this answers is not in the document]
Comment (iamrentman) [reply core/src/scss/core/fonts/_fonts.scss]: Apologies for the confusion...I was so excited that I flubbed the first go. 🤦 I've committed changes based on your feedback here and also updated the import of Source Sans to match the Serif import. Hope that is OK.

**Review by yvonnetangsu — APPROVED** [review]
This looks great - thank you for the update @iamrentman ! 😄 🎉 

One optional request - if you could please help us update the import lines for the roboto slab, noto sans and kalam fonts as well, that'd be great. Pre-approving since it's totally optional - we can do that later as well. We rarely use those other fonts anyway.

**Comment by iamrentman** [comment]
@yvonnetangsu I've committed updates to the additional font imports – Roboto, Kalam, and Noto Sans – to match what was there.

When selecting the styles to include for Noto Sans Devanagari, there are no italic versions available, so I only included 400 and 700: https://fonts.google.com/noto/specimen/Noto+Sans+Devanagari. I'm not familiar with this particular font so maybe there is another way to include the italics? Or maybe we're good the way I have it here? Let me know what you think.

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu I've committed updates to the additional font imports – Roboto, Kalam, and Noto Sans – to match what was there.
> 
> When selecting the styles to include for Noto Sans Devanagari, there are no italic versions available, so I only included 400 and 700: https://fonts.google.com/noto/specimen/Noto+Sans+Devanagari. I'm not familiar with this particular font so maybe there is another way to include the italics? Or maybe we're good the way I have it here? Let me know what you think.

@iamrentman Thank you - this is perfect. I don't think we support the italic version for Noto Sans/Kalam/Roboto Slab currently so this is great 😄 💯

**Comment by yvonnetangsu** [comment]
@sherakama Not sure why the autorelease failed - I'm guessing it's because of the PR labeler failed for some reason? 🤔

**Comment by sherakama** [comment]
Probably missing a key or a permission setting...

Screen Shot 2022-09-14 at 10 57 16 AM

**Comment by sherakama** [comment]
There you go: https://github.com/SU-SWS/decanter/releases/tag/6.2.9

**Pull request merged**